.nvmrc

```diff
@@ -0,0 +1 @@
+16
```

Comment:
@neubee I added this file to be (somewhat) explicit about the version of node used by cloudflare. I'm guessing you currently have `NODE_VERSION` set in the cloudflare configuration, which has the same effect. Note that svelte-kit 1.0.1 explicitly doesn't support node 17 (as it isn't an LTS version) and cloudflare pages doesn't support node 18.

Reply:
Nice, I did not know this was something you could do.